internal/command/jsonformat/renderer.go

```diff
@@ -240,6 +264,49 @@ func (r Renderer) RenderHumanPlan(plan Plan, mode plans.Mode, opts ...RendererOp
 	}
 }
 
+func (r Renderer) RenderLog(log *JSONLog) error {
+	switch log.Type {
+	case LogApplyStart, LogApplyComplete, LogRefreshStart:
+		msg := fmt.Sprintf("[bold]%s[reset]", log.Message)
+		r.Streams.Println(r.Colorize.Color(msg))
```

Comment:
```suggestion
		r.Streams.Printf(r.Colorize.Color("[bold]%s[reset]"), log.Message)
```

As a rule, we must not colorize an interpolated string, as it means that any instances of color codes (e.g. `[red]`) inside user-defined content will be incorrectly rendered. Instead, we can colorize the format string, then pass that result to `Printf`.